x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/analytics/100_tsdb.yml

```diff
@@ -85,6 +85,32 @@ aggretate multi_terms:
   - match: { aggregations.m_terms.buckets.2.key_as_string: "{k8s.pod.uid=947e4ced-1786-4e53-9e0c-5c447e959507, metricset=pod}|10.10.55.2" }
   - match: { aggregations.m_terms.buckets.2.doc_count: 1 }
 
+---
+"multi_terms aggregation with time_series aggregation":
+  - skip:
+      version: " - 8.12.99"
+      reason: "multi_terms for time series aggregation fixed in 8.13.0"
+
+  - do:
+      search:
+        index: test
+        body:
+          aggs:
+            ts:
+              time_series: {}
+            m_terms:
+              multi_terms:
+                collect_mode: breadth_first
```

Comment:
Should add a test where we check that execution of multi terms fails if the user specifies collect_mode: breadth_first and uses a time series aggregation together with a multi terms aggregation?